### PR TITLE
Adding new state selectors

### LINKED ISSUE #3466: Update State Selection docs to add `unmodified` and `old`
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### What page(s) or areas on docs.getdbt.com are affected?

https://docs.getdbt.com/reference/node-selection/methods#the-state-method

### What changes are you suggesting?

docs improvement related to an open dbt-core PR [#7773](https://github.com/dbt-labs/dbt-core/pull/7773). This issue will simply be to add two new state selectors to the docs.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding the `state:unmodified` and `state:old` selectors

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Comment by jtcohen6** [comment]
@matthewshaver https://github.com/dbt-labs/dbt-core/pull/7773 hasn't been reviewed/merged yet — let's hold off on merging this docs update until we know when it will be released

**Update:** It's been merged! This PR is cleared for landing

File: website/docs/reference/node-selection/methods.md [hunk 1]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
 
 Remember that `state:modified` includes _all_ of the criteria above, as well as some extra resource-specific criteria, such as modifying a source's `freshness` or `quoting` rules or an exposure's `maturity` property. (View the source code for the full set of checks used when comparing [sources](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L660-L681), [exposures](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L768-L783), and [executable nodes](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L319-L330).)
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old`: a node with the same `unique_id` exists in the comparison manifest
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
- `state:old` &mdash; a node with the same `unique_id` exists in the comparison manifest
```

File: website/docs/reference/node-selection/methods.md [hunk 2]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
 
 Remember that `state:modified` includes _all_ of the criteria above, as well as some extra resource-specific criteria, such as modifying a source's `freshness` or `quoting` rules or an exposure's `maturity` property. (View the source code for the full set of checks used when comparing [sources](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L660-L681), [exposures](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L768-L783), and [executable nodes](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L319-L330).)
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old` &mdash; a node with the same `unique_id` exists in the comparison manifest
+- `state:unmodified`: All existing nodes with no changes
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
- `state:unmodified` &mdash; All existing nodes with no changes 
```

File: website/docs/reference/node-selection/methods.md [hunk 3]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
...
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old` &mdash; a node with the same `unique_id` exists in the comparison manifest
+- `state:unmodified` &mdash; All existing nodes with no changes 
+
+The benefits for these selectors are primarily to exclude unchanged nodes to shorten run times. There are no subselectors available, but that may change as use cases evolve.
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
The primary benefit of these selectors is to exclude unchanged nodes to shorten run times. No subselectors are available at this time, but that may change as use cases evolve. 
```

File: website/docs/reference/node-selection/methods.md [hunk 4]
```diff
@@ -178,7 +178,7 @@ The `state` method is used to select nodes by comparing them against a previous
 
 </VersionBlock>
 
-`state:new`: There is no node with the same `unique_id` in the comparison manifest
+`state:new`: There is no node with the same `unique_id` in the comparison manifest.
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
`state:new`: There is no node with the same `unique_id` in the comparison manifest
```

File: website/docs/reference/node-selection/methods.md [hunk 5]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
 
 Remember that `state:modified` includes _all_ of the criteria above, as well as some extra resource-specific criteria, such as modifying a source's `freshness` or `quoting` rules or an exposure's `maturity` property. (View the source code for the full set of checks used when comparing [sources](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L660-L681), [exposures](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L768-L783), and [executable nodes](https://github.com/dbt-labs/dbt-core/blob/9e796671dd55d4781284d36c035d1db19641cd80/core/dbt/contracts/graph/parsed.py#L319-L330).)
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old` &mdash; a node with the same `unique_id` exists in the comparison manifest
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
- `state:old` &mdash; A node with the same `unique_id` exists in the comparison manifest
```

File: website/docs/reference/node-selection/methods.md [hunk 6]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
...
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old` &mdash; A node with the same `unique_id` exists in the comparison manifest
+- `state:unmodified` &mdash; All existing nodes with no changes 
+
+The primary benefit of these selectors is to exclude unchanged nodes to shorten run times. No subselectors are available at this time, but that may change as use cases evolve.
```
Comment (nghi-ly) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
These selectors can help you shorten run times by excluding unchanged nodes. Currently, no subselectors are available at this time. 
```

**Review by nghi-ly — COMMENTED** [review]
looks good! left one minor sugg

File: website/docs/reference/node-selection/methods.md (hunk 6 above)
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
The selectors can help you shorten run times by excluding unchanged nodes. Currently, no subselectors are available at this time, but that may change as use cases evolve. 
```

File: website/docs/reference/node-selection/methods.md [hunk 7]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
...
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old` &mdash; A node with the same `unique_id` exists in the comparison manifest
+- `state:unmodified` &mdash; All existing nodes with no changes 
+
+The selectors can help you shorten run times by excluding unchanged nodes. Currently, no subselectors are available at this time, but that may change as use cases evolve.
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
These selectors can help you shorten run times by excluding unchanged nodes. Currently, no subselectors are available at this time, but that may change as use cases evolve. 
```

File: website/docs/reference/node-selection/methods.md [hunk 8]
```diff
@@ -200,6 +200,16 @@ Because state comparison is complex, and everyone's project is different, dbt su
...
 
+<VersionBlock firstVersion="1.6">
+
+There are two additional `state` selectors that complement `state:new` and `state:modified` by representing the inverse of those functions:
+- `state:old` &mdash; A node with the same `unique_id` exists in the comparison manifest
+- `state:unmodified` &mdash; All existing nodes with no changes 
+
+These selectors can help you shorten run times by excluding unchanged nodes. Currently, no subselectors are available at this time, but that may change as use cases evolve.
```
Comment (nghi-ly) [inline website/docs/reference/node-selection/methods.md]:
```suggestion
These selectors can help you shorten run times by excluding unchanged nodes. Currently, no subselectors are available at this time, but that might change as use cases evolve. 
```

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**